content/docs/gke/deploy/deploy-ui.md

```diff
@@ -51,4 +47,60 @@ takes longer than expected, try accessing the above URI anyway.
   [upgrade or reinstall a Kubeflow Pipelines 
   deployment](/docs/pipelines/upgrade/).
 * [Troubleshoot](/docs/gke/troubleshooting-gke) any issues you may
-  find.
\ No newline at end of file
+  find.
+  
+  <!--Sub-sections, these will be not be part of this page  -->
```

Comment:
I'm wondering why these subsections shouldn't be part of this page? My inclination is to keep them on the same page, to make them easier to find.

Reply:
Ok, I thought the page will look cluttered. Can we make them like a drop down? to be opened only when needed?

Reply:
Our Hugo theme doesn't currently have the facility to hide text behind a dropdown. Actually, UI experts say dropdowns of that nature (where content is hidden) don't perform well in UX testing. If the page is well organized, it should be OK to have all this info on one page. May people prefer long pages where necessary to hold all the relevant info, rather than having to click to a new page then back again.